agent/agent_test.go

```diff
@@ -4524,6 +4527,9 @@ LOOP:
 }
 
 // This is a mirror of a similar test in agent/consul/server_test.go
+//
+// TODO(rb): implement something similar to this as a full containerized test suite with proper
```

Comment:
Calling out this TODO, since I could not find a way to do the "firewalling" of the servers from each other in-process.